src/main/java/zoe/Event.java

```diff
@@ -0,0 +1,38 @@
+package zoe;
+
+/**
+ * Subclass of task
+ * Creates an event when keyed in the form: event XYZ /from 123 /to 123
+ */
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
Would it be better to state explicitly that the two strings represent timestamp?